docs/installation/integrations/next-js.md

```diff
@@ -0,0 +1,204 @@
+---
+menu-title: Next.js
+meta-title: Compatibility with Next.js | CKEditor 5 documentation
+meta-description: Learn how to integrate the Next.js framework with the rich text editor - CKEditor 5 - using both the App Router and Pages Router routing strategies.
```

Comment:
I would move CKEditor a bit more to the beginning of the sentence to be more noticeable.

App Router and Pages Router are exclusive, so I would add `or`.

> Learn how to integrate CKEditor 5  with the Next.js framework using the App Router or a Pages Router routing strategies.